src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -95,11 +107,15 @@ def register_path_argument(scope, default_file_param=None, options_list=None):
 directory_type = CliArgumentType(options_list=('--directory-name', '-d'), help='The directory name.', completer=get_storage_name_completion_list(FileService, 'list_directories_and_files', parent='share_name'))
 file_name_type = CliArgumentType(options_list=('--file-name', '-f'), completer=get_storage_name_completion_list(FileService, 'list_directories_and_files', parent='share_name'))
 share_name_type = CliArgumentType(options_list=('--share-name', '-s'), help='The file share name.', completer=get_storage_name_completion_list(FileService, 'list_shares'))
+table_name_type = CliArgumentType(options_list=('--table-name', '-t'), completer=get_storage_name_completion_list(TableService, 'list_tables'))
+queue_name_type = CliArgumentType(options_list=('--queue-name', '-q'), help='The queue name.', completer=get_storage_name_completion_list(QueueService, 'list_queues'))
+IGNORE_TYPE = CliArgumentType(help=argparse.SUPPRESS, nargs='?', action=IgnoreAction, required=False)
 
```

Comment:
I'll move it. It is really only part of the solution to 675 in that it helps standardize the way in which ignored arguments handled. It does not fully address that issue though because it doesn't affect tab completion, so that's why I left it here for now.

---
In reply to: [74456470](https://github.com/Azure/azure-cli/pull/681#discussion_r74456470) [](ancestors = 74456470)